conf/application.conf

```diff
@@ -67,10 +67,25 @@ microservice {
       host     = localhost
       port     = 16002
     }
+
+    returns {
+      protocol = http
+      host     = localhost
+      port     = 16004
+      clientId = "returns stub client id"
+      secret   = "Shh! It's a secret!"
+      url {
+        getReturn = "/RESTAdapter/EXCISE/Return/{0}/{1}/{2}"
```

Comment:
Latest API suggests this is /RESTAdapter/excise/{ad}/return/{appaId}/{periodKey}

Reply:
Update - it is /etmp/RESTAdapter .....

Reply:
Fixed